src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random randomNumber = new Random();
+        Random randomValueOfField = new Random();
+        final int upperValueBound = 1000;
+        final int amountOfShapes = 4;
+        int shapeCase = randomNumber.nextInt(amountOfShapes) + 1;
+        int randomValue = randomValueOfField.nextInt(upperValueBound) + 1;
+        switch (shapeCase) {
+            case 1:
+                Square square = new Square(ColorSupplier.getRandomColor(), randomValue);
+                return square;
+            case 2:
+                Rectangle rectangle = new Rectangle(ColorSupplier.getRandomColor(), randomValue,
+                        randomValue);
```

Comment:
Зроби щоб у фігур параметри ініціалізувалися різними значеннями
```suggestion
                Rectangle rectangle = new Rectangle(ColorSupplier.getRandomColor(), firstRandomValue,
                        secondRandomValue);
```